pyroma/ratings.py

```diff
@@ -18,6 +18,7 @@
 import re
 from collections import defaultdict
 
+import packaging
 from docutils.core import publish_parts
 from docutils.utils import SystemMessage
 from trove_classifiers import classifiers as CLASSIFIERS
```

Comment:
The `specifiers` submodule needs to be imported explicitly here:

```suggestion
import packaging.specifiers
from docutils.core import publish_parts
from docutils.utils import SystemMessage
from trove_classifiers import classifiers as CLASSIFIERS
```